logs/all.log

```diff
@@ -0,0 +1,42 @@
+2022-08-09 22:08:08,945 DEBUG m.j.c.s.r.c.Main [main] Some debug log
```

Comment:
We don't add. logs to github, add all logs to .gitignore
We also don't want to see target folder in github, add it to .gitignoree